### PR TITLE
Added instructions for EC3 developers

### PR DESCRIPTION
**Pull request by glarocca**
# Summary

---

**Related issue :**

### CODE REVIEW

File: content/en/users/cloud-compute/ec3/developers/_index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Instructions for developers"
+type: docs
+weight: 30
+description: >
+  How to integrate a new application in EC3
+---
+
+This section documents how to integrate a new application in EC3.
+
```
Comment (gwarf) [inline content/en/users/cloud-compute/ec3/developers/_index.md:10]:
```suggestion
```

File: content/en/users/cloud-compute/ec3/developers/_index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+The process to integrate a new application in EC3 is described by the 
+following process:
+
+* Describe the application to be integrated with Ansible, the open-source 
+  automation engine that automates software provisioning, configuration 
+  management, and application deployment.
+* Use the Ansible receipt to create a new RADL template
+
+### Documentations
```
Comment (gwarf) [inline content/en/users/cloud-compute/ec3/developers/_index.md:21]:
```suggestion
### Documentations

```

File: content/en/users/cloud-compute/ec3/developers/_index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+* [Ansible documentation](http://docs.ansible.com/)
+* [EC3 documentation](http://ec3.readthedocs.io/en/devel/templates.html)
+* [RADL](https://github.com/grycap/ec3/tree/master/templates)
+
+### Contacts
```
Comment (gwarf) [inline content/en/users/cloud-compute/ec3/developers/_index.md:27]:
```suggestion
### Contacts

```

File: content/en/users/cloud-compute/ec3/developers/_index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+* Amanda Calatrava at: amcaar <at> i3m <dot> upv <dot> es
+To deploy [NAMD](https://www.ks.uiuc.edu/Research/namd/) clusters, please
+select one of the available LRMS (Local Resource Management System) and
+choose NAMD from the list of applications.
+
+## How to use generic tools/practices in EC3
+
+### ECAS cluster
+
+Check the dedicated [ECAS documentation](./ecas/).
```
Comment (gwarf) [inline content/en/users/cloud-compute/ec3/developers/_index.md]: Broken link: https://github.com/EGI-Foundation/documentation/runs/1523150242?check_suite_focus=true#step:4:52

File: content/en/users/cloud-compute/ec3/developers/_index.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+* [ImageMagick](https://www.imagemagick.org/), a tool to manipulate images.
+* [Radiomics](https://github.com/eubr-atmosphere/radiomics), a use case about
+  the handling of Rheumatic Heart Disease (RHD) through image computing and
+  Artificial Intelligence (AI).
+
+### SLURM cluster
+
+To deploy [SLURM](https://slurm.schedmd.com/documentation.html) clusters,
+please select SLURM from the list of available LRMS.
+See also the dedicated guide on [HTC clusters](./htc/)
```
Comment (gwarf) [inline content/en/users/cloud-compute/ec3/developers/_index.md]: Broken link: https://github.com/EGI-Foundation/documentation/runs/1523150242?check_suite_focus=true#step:4:53

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks!
I've added a few suggestions and there are two links to be fixed.

**Review by gwarf — APPROVED** [review]

**Review by enolfc — APPROVED** [review]
LGTM!

We need to have some discussion about the AoD / EC3 docs, I feel they probably need to be rearranged

**Pull request merged**